### DISCUSSION TITLE
Aqara Hub Pair into Zigbee2MQTT?

### DISCUSSION THREAD

**Discussion by stokes776** [Koenkk/zigbee2mqtt#10778] [Devices]
Hello,

I have an Aqara Hub that I purchased and setup before I entered the world of Zigbee2MQTT.
The hub has an LED on it that is kind of nice to be able to control. I use it with a Single Door sensor. When the door sensor is OPEN the LED lights up.

Is there a way to get that Aqara Hub to JOIN the Zigbee2MQTT network so that I can see its values on MQTT and control its LED?

https://www.amazon.com/gp/product/B08BZQNK4L/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

**Comment by Koenkk** [comment]
This is not possible, a coordinator (which the Aqara hub is) cannot join another network.

**Comment by stokes776** [comment]
Thank you @Koenkk . Just out of curiosity, is there a name for a device that would do something similar, easily plug into an outlet, and just have a small LED indicator right that I can control via MQTT?

I know I can make one. I know an LED light bulb is basically the same thing, but thinking a little more form factor purpose built already existing type of device.

**Reply by Koenkk** [reply]
I'm not aware of any
